### DISCUSSION TITLE
make:scaffold

### DISCUSSION THREAD

**Discussion by bitsmyth** [laravel/framework#51434] [Ideas]
What do you think about the idea of extending the `php artisan make` command palette to include an overall opinionated scaffold to execute multiple commands under the hood. I propose this idea to speed up development by adding desired attributes like `php artisan make:scaffold Car tires:int color:references` to skip repetitive boilerplating even further. So I could imagine that the scaffold command creates the model, migration, controller, view & the according tests.

We currently don't have any way besides of basic creation of skeletons - which is a pity imho.

**Comment by ghost** [comment]
Hi. I can see that codding is transforming into running terminal commands more and more in laravel:)

PROS:
- faster development time in theory
- some devs will be happier

CONS:
- you need to change all auto generated code if you follow PSR12 for example (or if your codding style is not the same as laravel's one)
- you occupy your mind with commands instead of code

An alternative is copy paste&edit codding.
